package.json

```diff
@@ -49,36 +49,36 @@
 		"bench:watch": "vitest bench"
 	},
 	"dependencies": {
-		"@prismicio/custom-types-client": "^1.1.0",
+		"@prismicio/custom-types-client": "^1.3.1",
 		"common-tags": "^1.8.2",
-		"fast-glob": "^3.2.12",
-		"jiti": "^1.18.2",
-		"joi": "^17.9.2",
-		"meow": "^12.0.1",
-		"node-fetch": "^3.3.1",
+		"fast-glob": "^3.3.3",
+		"jiti": "^1.21.7",
+		"joi": "^17.13.3",
+		"meow": "~12.0.1",
```

Comment:
❓ What is the explanation behind the use of `~` vs `^`?

Reply:
Version 12.1.0 didn't work for some reason :(

Reply:
Got it 🙂